### PR TITLE
TiUP: fix grammar and format for topology.example.yaml

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

1. Fix grammar mistakes
2. Unify format

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/pingcap-incubator/tiup-cluster/pull/298

### CODE REVIEW

**Comment by yikeke** [comment]
Please also update the file template in [quick start guide doc](https://pingcap.com/docs-cn/dev/quick-start-with-tidb/). @toutdesuite

**Comment by toutdesuite** [comment]
@yikeke The file is updated. PTAL

File: how-to/deploy/orchestrated/tiup.md [hunk 1]
```diff
@@ -1280,7 +1278,7 @@ tidb_servers:
     deploy_dir: "deploy/tidb-4000"
     log_dir: "deploy/tidb-4000/log"
     numa_node: "0,1"
-    # Config is used to overwrite the `server_configs.tidb` values
+    # The following configs are used to overwrite the `server_configs.pd` values.
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:1281]:
```suggestion
    # The following configs are used to overwrite the `server_configs.tidb` values.
```

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -185,9 +185,8 @@ TiDB-Wasm 是运行在浏览器中的 TiDB 数据库，打开网页即可使用
     {{< copyable "shell-regular" >}}
     
     ```yaml
-    # # Global variables are applied to all deployments and as the default value of
-    # # them if the specific deployment value missing.
-     
+    # # Global variables are applied to all deployments and used as the default value of
+    # # the deployments if a specific deployment value is missing.
```
Comment (yikeke) [inline quick-start-with-tidb.md:189]: Line 196 needs to be updated too.

**Pull request merged**